biz.aQute.repository/test/aQute/bnd/repository/maven/pom/provider/PomRepositoryTest.java

```diff
@@ -714,12 +714,17 @@ public void testMultipleRevisions() throws Exception {
 			assertFalse(resources.isEmpty());
 
 			assertThat(mcsr.list("*")).contains("biz.aQute.remote.launcher",
-				"biz.aQute.launcher", "biz.aQute.junit", "biz.aQute.tester", //
-				"org.jacoco:org.jacoco.agent", // with classifier
+				"biz.aQute.launcher", "biz.aQute.junit", "biz.aQute.tester",
+				// "org.jacoco:org.jacoco.agent:0.8.6"
+				// a bundle so normal bsn
+				"org.jacoco.agent",
+				// "org.jacoco:org.jacoco.agent:jar:runtime:0.8.6"
+				// with classifier & not a bundle
+				"org.jacoco:org.jacoco.agent:jar:runtime",
```

Comment:
This is changing the pseudo-bsn to include extension and classifier. It currently is groupId:artifactId only.

By changing the pseudo-bsn to include extension and classifier, you will break some builds which don't include this. You can see this here in the needed change in the test case. The pseudo-bsn of `org.jacoco:org.jacoco.agent` had to be changed to `org.jacoco:org.jacoco.agent:jar:runtime`.

I am not sure such a breaking change is a good idea.

Reply:
Afaik we did bit release a version where a gav with an classifier will be processed from pomrepository.

So this is mor a fix. 
Though that this is now the same behavior like the bndmaven plugin and the pom plugin when it uses a pom and not a revision.